ingestion/src/metadata/profiler/metrics/hybrid/histogram.py

```diff
@@ -119,7 +119,13 @@ def _get_bins(
         if res_iqr is not None:
             # freedman-diaconis rule
             bin_width = self._get_bin_width(float(res_iqr), res_row_count)  # type: ignore
-            num_bins = math.ceil((res_max - res_min) / bin_width)  # type: ignore
+            try:
+                num_bins = math.ceil((res_max - res_min) / bin_width)  # type: ignore
+            except OverflowError as warning:
+                logger.warn(
+                    f"An warning occurred while computing the number of bins: {warning}"
+                )
+                num_bins = max_bin_count
```

Comment:
Actually looking at this piece of code we won't be able to do anything if min/max is inf as we need it to compute the width of the bins. For now we can skip the histogram if we fall into this.